docs/devops.md

```diff
@@ -93,20 +88,21 @@ Terraform is an infrastructure software as code, developed by HashiCorp. It enab
 
 * * *
 
-### Introduction to HashiCorp Terraform 
+## Introduction to HashiCorp Terraform 
 
 <figure class="video_container">
   <iframe width="100%" height="315" src="https://www.youtube.com/embed/h970ZBgKINg" frameborder="0" allowfullscreen="true">
   </iframe>
 </figure>
 
-## Introduction to Docker and Kubernetes 
+## Infraestructura como código en EOS CR
```

Comment:
Why "Infraestructura como código en EOS CR"?

Reply:
Thank you very much for your observation